jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce-gci.yaml

```diff
@@ -88,6 +88,7 @@
                 export GINKGO_TEST_ARGS="--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"
                 export GINKGO_PARALLEL="y"
                 export PROJECT="e2e-gce-gci-ci-master"
+                export KUBE_MASTER_OS_DISTRIBUTION="gci"
```

Comment:
Sorry, I thought we were only testing GCI masters. Added KUBE_NODE_OS_DISTRIBUTION as well.